nbcparser/main.go

```diff
@@ -1,40 +1,36 @@
 package main
 
 import (
-	"log"
+	"fmt"
 	"os"
 
 	"github.com/Azure/agentbaker/nbcparser/pkg/parser"
 )
 
-// input to this function will be the serialized JSON from userdata + custom data.
+// This script will take in the bootstrap contract in JSON format,
 // it will be deserialized to the contract that the VHD this binary will be on supports.
 // Parse will be called using that deserialized struct and output the generated cse_cmd to trigger the bootstrap process.
-// example usage:
-// to build: go build main.go.
-// to run: ./main testdata/test_nbc.json.
 //
 //nolint:gosec // generated cse_cmd.sh file needs execute permissions for bootstrapping
+const (
+	CSE_CMD = "cse_cmd.sh"
+)
+
 func main() {
 	if len(os.Args) < parser.MinArgs {
-		log.Default().Printf("Usage: %s <input.json>", os.Args[0])
-		return
+		fmt.Printf("Usage: %s <input.json>", os.Args[0])
+		os.Exit(1)
 	}
-
 	// Read in the JSON file
 	inputJSON, err := os.ReadFile(os.Args[1])
 	if err != nil {
-		log.Fatal(err)
-		return
+		fmt.Println(err)
+		os.Exit(1)
 	}
-
 	cseCmd, err := parser.Parse(inputJSON)
 	if err != nil {
-		log.Fatal(err)
-		return
-	}
-	if err = os.WriteFile("cse_cmd.sh", []byte(cseCmd), 0655); err != nil {
-		log.Fatal(err)
-		return
+		fmt.Println(err)
+		os.Exit(1)
 	}
+	fmt.Println(cseCmd)
```

Comment:
do we still need this?

Reply:
Yes, I was printing the generate cse_cmd to stdout and then piping it into a file to be executed in the systemd unit. However, I decided to just trigger cse_cmd from the go binary itself, refer to recent commits.